shared/elements/gaia-header/dist/gaia-header.js

```diff
@@ -531,14 +531,19 @@ var base = window.GAIA_ICONS_BASE_URL
   || window.COMPONENTS_BASE_URL
   || 'bower_components/';
 
-// Load it if it's not already loaded
-if (!isLoaded()) { load(base + 'gaia-icons/gaia-icons.css'); }
+if (!document.documentElement) {
+  window.addEventListener('load', load);
```

Comment:
It was added so that gaia-icons could be loaded in add-ons before `document.head` exists. Perhaps `DOMContentLoaded` would be better.